nltk/lm/api.py

```diff
@@ -162,6 +162,8 @@ def context_counts(self, context):
 
     def entropy(self, text_ngrams):
         """Calculate cross-entropy of model for given evaluation text.
+        This implementation is based on the Shannon-McMillan-Breiman theorem,
+        as used and referenced by Jurafsky and Jordan Boyd-Graber.
```

Comment:
```suggestion
        as used and referenced by Dan Jurafsky and Jordan Boyd-Graber.
```
Consistently use full names